zoo/base/settings.py

```diff
@@ -36,6 +37,13 @@
     ZOO_AUDITING_DROP_ISSUES=(int, 7),
     ZOO_SONARQUBE_URL=(str, None),
     ZOO_SONARQUBE_TOKEN=(str, None),
+    AWS_CONFIG=(str, None),
+    AWS_CONFIG_FILE=(str, "/tmp/aws/config"),
+    AWS_SHARED_CREDENTIALS=(str, None),
+    AWS_SHARED_CREDENTIALS_FILE=(str, "/tmp/aws/credentials"),
+    RANCHER_API_URL=(str, None),
+    RANCHER_USERNAME=(str, None),
```

Comment:
Username and password? Shouldn't it be `RANCHER_ACCESS_KEY` and `RANCHER_SECRET_KEY`?

Reply:
Yes, it should. I'll fix it.